src/python/setup.cfg

```diff
@@ -18,7 +18,7 @@ classifiers =
     Operating System :: Unix
     Intended Audience :: Developers
     Intended Audience :: Science/Research
-    Intended Audience :: Other Audience
+    Intended Audience :: Any other Audience
```

Comment:
Hi!
Thank you for your suggested improvement
However, I think the original classifier was correct based on [the list of possible values in PyPi](https://pypi.org/classifiers/)

Feel free to see if there are some others we could add to this list though!